swarm/network/protocol_test.go

```diff
@@ -92,30 +112,36 @@ func newBzzBaseTester(t *testing.T, n int, prvkey *ecdsa.PrivateKey, spec *proto
 	record.Set(NewENRAddrEntry(bzzKey))
```

Comment:
you don't need all this, in this case the hive has already been created before, so this is just recreating a (later un-used) new one